gitcommands.go

```diff
@@ -423,6 +423,54 @@ func gitSquashPreviousTwoCommits(message string) (string, error) {
 	return runDirectCommand("git reset --soft HEAD^ && git commit --amend -m \"" + message + "\"")
 }
 
+func gitRebaseTwoCommits(branchName string, shaValue string, action string) (string, error) {
+	//	return runDirectCommand("GIT_SEQUENCE_EDITOR='sed -i \"s/^pick " + shaValue + "/" + action + " " + shaValue + "/g\" $1' git rebase -i " + shaValue + "~2")
+	// git chekout -b name-hash-temp-branch
+	// git reset --hard shaValue
+	// git commit --amend --fixup=shaValue^
+	// git checkout name
+	// git rebase name-hash-temp-branch
+	// git branch -D name-hash-temp-branch
+	// GIT_SEQUENCE_EDITOR="" git rebase -i --autosquash
+	ret := "" + branchName + shaValue + "\n"
+	output, err := runDirectCommand("git checkout -b " + branchName + "-" + shaValue + "-temp-branch")
```

Comment:
This can be DRY'd up by defining an array of strings (each argument to runDirectCommand), then iterating over the array and saying
```
commands := []string{
	"command 1",
	"command 2"
}
for _, command := range commands {
	output, err := runDirectCommand(command)
	ret += output
	if err != nil {
		return ret, err
	}
}
```